pulser/simulation/simulation.py

```diff
@@ -241,7 +267,29 @@ def make_vdw_term() -> float:
                 vdw += U * self._build_operator('sigma_rr', q1, q2)
             return vdw
 
-        def build_coeffs_ops(basis: str, addr: str) -> list[list]:
+        def make_interaction_xy_term():
+            """Construct the XY interaction Term.
+
+            For each pair of qubits, calculate the distance between them,
+            then assign the local operator "sigma_du * sigma_ud" at each pair.
+            The units are given so that the coefficient
+            includes a 1/hbar factor.
+            """
+            xy = 0
+            # Get every pair without duplicates
+            for q1, q2 in itertools.combinations(self._qdict.keys(), r=2):
+                dist = np.linalg.norm(
+                    self._qdict[q1] - self._qdict[q2])
+                cosine = 0.
+                U = 0.5 * 3700 * \
```

Comment:
This value should not be hard-coded here, but rather specified in the device, as is the case for `C_6`.